static/index.html

```diff
@@ -51,26 +53,24 @@
         <main class="normalize" id="grapheneos">
             <div class="content hero">
 
-                <div>
+                <div class="hero-text">
                     <h1><a href="#grapheneos">GrapheneOS</a></h1>
-                    <p>The private and secure mobile operating system with Android app compatibility.
-                        Developed as a non-profit open source project.</p>
-                    <a class="button" href="/install/">Install GrapheneOS</a>
+                    <p>Privacy and security, crafted for your mobile device, usable by everyone.</p>
+                    <a id="install-button" class="button" href="/install/">Install Now</a>
+                    <p class="hero-little">
+                        Available for Google Pixel 4 and newer Google Pixel phones.
```

Comment:
Pixel 3 are still supported, according to https://grapheneos.org/build :)

Reply:
You're right. I'll fix that.